### ISSUE TITLE
Return of repository to the creator

### ISSUE THREAD

**Issue by Codcore** [Codcore/Amethyst#81]
Hi to all members of crystal-community! I am an creator of Amethyst framework, and I passed it to crystal-community in hope that here this project  will be develop. But, as I see there are no changes to the project and it seems dead. I I demand to return it to my hands, I have people that want to take care of this project.

**Comment by veelenga** [comment]
I agree. Please be free to move it. Sorry for not being working on it for a while.

**Comment by bararchy** [comment]
@Codcore @veelenga .

I would suggest against it, before saying my reason I would like to say one thing, @Codcore it's your repo, it's your project, if you wanted it back it's right and you should take it, i'm not against it, I just have a comment regarding this move.

My reason is this, the Crystal-Community org, is there to allow projects to have a home where multiple contributors and admins are looking over the projects, it's by no means a "well take your project and continue it for you" but more of "if someone want's to contribute, we will make it easier for them by having faster PR\comment cycles".

If you have people who want to work on Amethyst you should send them here, so that when you are away or have no time for the project, they can still merge and comment on PRs and let the project live on.

This is a bridge for contribution, and taking away the repo would most likely put it back in the state of abendonware without active development.

**Comment by sn** [comment]
Hi All,

I'm the one that wants to take over Amethyst so I thought I'd add my comment here for clarification and to keep everyone satisfied. 

I've been working on Crystal since the earliest public releases and have it running in substantially large B2B and B2C products online (+- 600k active users). 

I also co-own a [software studio](https://wixehq.com) (12 years now). 

We have big plans to keep Amethyst alive and well, up to date. This is a commitment I can publicly make and be accountable for. It will always be public, but fall into our development processes and release cycles. 

Perhaps one day we can put it back into crystal-community, but for now myself and my team want to do our bit and bring the project back to life. 

I'll be happy to share the roadmap and action plan for the next few months on the project with anyone that wants to know. 

sean@wixelhq.com

**Comment by bararchy** [comment]
@SeanNieuwoudt this is a different case all together, I have no objection and your argument makes sense.
We do that same with SHAInet in NeuraLegion, as we need to to be in par with our own internal goals.

Go and make Amethyst great again ;)

**Comment by sn** [comment]
@bararchy I'm a big fan of NeuraLegion btw. 

Thanks!

**Comment by bararchy** [comment]
Hi @SeanNieuwoudt , how goes the work on Amethyst?

**Comment by sn** [comment]
@bararchy It's going really well at the moment. I'm doing it solo and will push through everything soon. Waiting for the design of the new website and documentation to come through at the moment. I've re-written from the ground up to bring it on par with other frameworks in terms of performance and usability.

**Comment by renich** [comment]
Any updates on this one @sn?

**Comment by sn** [comment]
@renich Yeah, I've been working on this quite a lot in private and will release within the next 60 days or so. I wanted to first flesh it out on a real system with real users so we're running it on a large subset of the indiemaker.co Rails app and incrementally making improvements.

**Comment by renich** [comment]
Awesome news!

**Comment by vinyll** [comment]
@sn any wip to be shared in a branch eventually?
